### PR TITLE
level agnostic experiment and improved evaluation for level agnostic, level1 & historical

### PR DESCRIPTION
**Pull request by ellieking17**

### CODE REVIEW

File: python/dataprep.py [hunk 1]
```diff
@@ -25,42 +25,83 @@
 METADATA_LIST = os.getenv('METADATA_LIST').split()
 
 
-# **** RESHAPE data long -> wide by taxon *******
-# ***********************************************
+def load_labelled(SINCE_THRESHOLD, level='level2'):
+    if level=='agnostic' or level=='level1':
```
Comment (cbaines) [inline python/dataprep.py]: It might be neater to write this as `level in ('agnostic', 'level1')`.

File: python/dataprep.py [hunk 2]
```diff
@@ -25,42 +25,83 @@
...
+        os.path.join(DATADIR, 'labelled.csv.gz'),
+        dtype=object,
+        compression='gzip'
+         )
+    else:
+        dataframe = pd.read_csv(
+        os.path.join(DATADIR, 'labelled_level2.csv.gz'),
+        dtype=object,
+        compression='gzip'
+    )
```
Comment (cbaines) [inline python/dataprep.py]: The indentation is a bit inconsistent here. Also, given that only the file parameter is different with these two `pd.read_csv` calls, you could perhaps just put the filename in a `if` statement, and move the `pd.read_csv` statement out.

File: python/dataprep.py [hunk 3]
```diff
@@ -25,42 +25,83 @@
...
 
-# reshape to wide per taxon and keep the combined text so indexing is consistent when splitting X from Y
+    if level=='level2':
+        # Create World taxon in case any items not identified
+        # through doc type in clean_content are still present
+        dataframe.loc[dataframe['level1taxon'] == 'World', 'level2taxon'] = 'world_level1'
 
+    if level=='level1' or level=='level2':
```
Comment (cbaines) [inline python/dataprep.py]: This could also be written as `level in ('level1', 'level2')`, which I think is a bit neater.

Comment (ellieking17) [reply python/dataprep.py]: Thanks - am making some further cahnges to this function for the next experiment so will add these style improvements then

**Pull request merged**